tests/integration/delete.bats

```diff
@@ -51,3 +51,58 @@ function teardown() {
   runc delete --force notexists
   [ "$status" -eq 0 ]
 }
+
+@test "runc delete --force in cgroupv2 with subcgroups" {
+  requires cgroups_v2 root
+  set_cgroups_path "$BUSYBOX_BUNDLE"
+
+  # grant `rw` priviledge to `/sys/fs/cgroup`
+  cat "${BUSYBOX_BUNDLE}/config.json"\
+   | jq '.mounts |= map((select(.type=="cgroup") | .options -= ["ro"]) // .)'\
+   > "${BUSYBOX_BUNDLE}/config.json.tmp"
+  mv "${BUSYBOX_BUNDLE}/config.json"{.tmp,}
+
+  # run busybox detached
+  runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+  [ "$status" -eq 0 ]
+
+  # check state
+  testcontainer test_busybox running
+
+  # create a sub process
+  __runc exec -d test_busybox sleep 1d
+  [ "$status" -eq 0 ]
+
+  # find the pid of sleep
+  pid=$(__runc exec test_busybox ps -a | grep 1d | awk '{print $1}')
+  [[ ${pid} =~ [0-9]+ ]]
+
+  # create subcgroups
+  cat <<EOF > nest.sh
+  cd /sys/fs/cgroup
+  for f in \$(cat cgroup.controllers); do echo +\$f > cgroup.subtree_control; done
```

Comment:
This gives an error on my system when trying to enable io controller.

Most probably just `+pids` is enough (and it won't error).